src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -365,7 +365,7 @@ export default function Swap({
       <SwapPoolTabs active={'swap'} />
       <AppBody className={className}>
         <SwapHeader />
-        <Wrapper id="swap-page">
+        <Wrapper id="swap-page" className={isExpertMode ? 'expertMode' : ''}>
```

Comment:
Classname used to apply conditional styling. Alternative is passing it as a prop to the styled component. This seemed the easiest/least complex.